Minji/02_배열제어.py

```diff
@@ -0,0 +1,11 @@
+def solution(arr):
+	for i in range(0, len(arr)):
+		if(arr.count(i) > 1):
```

Comment:
간단한 조건문의 경우 Python에서는 열고 닫는 괄호는 생략할 수 있어요!
```suggestion
		if arr.count(i) > 1:
```